publish.yml

```diff
@@ -141,7 +141,12 @@ jobs:
 
 - ${{ if coalesce(parameters.pypi_connection_name, parameters.artifact_feed) }}:
   - job: publish
-    dependsOn: ${{ parameters.targets }}
+
+    dependsOn:
+    - ${{ each target in parameters.targets }}:
+      - ${{ if or(startsWith(target, 'wheels_'), eq(target, 'sdist')) }}:
```

Comment:
Can this depend on any other jobs? Or would that just be the whole pipeline?

Reply:
Oh right I guess I don't need the second if statement since this already only goes over targets. But yeah not sure about depending on other jobs. I was just matching what was there before (targets)